rustls/src/server/server_conn.rs

```diff
@@ -561,6 +561,43 @@ impl From<ServerConnection> for crate::Connection {
     }
 }
 
+/// Unbuffered version of `ServerConnection`
+///
+/// See the [`crate::unbuffered`] module docs for more details
+pub struct UnbufferedServerConnection {
+    inner: UnbufferedConnectionCommon<ServerConnectionData>,
+}
+
+impl Deref for UnbufferedServerConnection {
+    type Target = UnbufferedConnectionCommon<ServerConnectionData>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.inner
+    }
+}
+
+impl DerefMut for UnbufferedServerConnection {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.inner
+    }
+}
+
+impl UnbufferedServerConnection {
+    /// Make a new ClientConnection. `config` controls how we behave in the TLS protocol, `name` is
+    /// the name of the server we want to talk to.
```

Comment:
I think this doc string isn't correct - there's no `name` argument and the connection _is_ the server.